dotnet/src/SemanticKernel.IntegrationTests/AI/OpenAICompletionTests.cs

```diff
@@ -42,12 +43,12 @@ public async Task OpenAITestAsync(string prompt, string expectedAnswerContains)
         OpenAIConfiguration? openAIConfiguration = this._configuration.GetSection("OpenAI").Get<OpenAIConfiguration>();
         Assert.NotNull(openAIConfiguration);
 
-        target.Config.AddOpenAICompletionBackend(
-            label: openAIConfiguration.Label,
+        target.Config.AddOpenAICompletion(
```

Comment:
nit: I would name it `AddOpenAITextCompletion`

Reply:
fixed - thanks!